myapp/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def new
+  end
+
+  def create
+    @new_task = Task.new(params.require(:task).permit(:title, :description))
+    @new_task.save
+      
+    flash[:success] = "New task was created successfully!"
```

Comment:
Did you run `bundle exec rubocop -a`? Anyway, single quotation is preferable
```suggestion
    flash[:success] = 'New task was created successfully!'
```